docs/sdk/auth/fragments/ios/custom-auth-flow.md

```diff
@@ -20,25 +18,31 @@ To initiate a custom authentication flow in your app, specify `authenticationFlo
   },
   "Auth": {
     "Default": {
+      // Add CUSTOM Authentication Flow
```

Comment:
Comments isn’t officially supported in JSON. https://stackoverflow.com/questions/244777/can-comments-be-used-in-json